app/App.tsx

```diff
@@ -1,19 +1,31 @@
 import React from "react";
 import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
+import AppLoading from "expo-app-loading";
+import { useFonts, Monda_700Bold } from "@expo-google-fonts/monda";
```

Comment:
Tady si nejsem jisty jak v react native vyresit custom fonty, moc se mi tohle reseni nelibi, ale nic lepsiho jsem nevygooglil.

Reply:
No úplně ideální to není, ale upravil bych to až zpětně. Teď se tím netrap.